packages/horizon/scripts/deploy.ts

```diff
@@ -0,0 +1,16 @@
+import { ignition } from 'hardhat'
```

Comment:
I think I initially added this because one of the contract calls during setup was reverting and I wanted to log some stuff, so I should probably remove it.

But now that I think of it, there is one potential use case where it might be handy. For our products and internal tooling we use a contract address book with a specific format which is different from the one ignition outputs. Changing everything to support ignition's format seems more complicated than just adding a post-deployment processing step to convert between formats. Though ... perhaps it's best to keep that post processing on an isolated hardhat task and run ignition through CLI.

Reply:
Maybe we could have a subtask that's run after a successful Ignition deployment, where you can run your data transformation. WDYT?

/cc @zoeyTM

Reply:
yeah that sounds great